### ISSUE TITLE
NPE @ Transform Selection (Subfiles)

### ISSUE THREAD

**Issue by nilsschmidt1337** [nilsschmidt1337/ldparteditor#291]
```
[LDPartEditor RELEASE 0.8.8 @ SWT v4527 OS win32]
java.lang.NullPointerException
	at org.nschmidt.ldparteditor.data.VM20Manipulator.transformSelection(VM20Manipulator.java:530)
	at org.nschmidt.ldparteditor.helpers.Manipulator.applyTranslation(Manipulator.java:939)
	at org.nschmidt.ldparteditor.helpers.composite3d.MouseActions.mouseUp(MouseActions.java:1170)
	at org.nschmidt.ldparteditor.composites.Composite3D$52.handleEvent(Composite3D.java:1213)
	at org.eclipse.swt.widgets.EventTable.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Widget.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:825)
	at org.eclipse.jface.window.Window.open(Window.java:801)
	at org.nschmidt.ldparteditor.shells.editor3d.Editor3DWindow.run(Editor3DWindow.java:5335)
	at org.nschmidt.ldparteditor.splash.SplashScreen.run(SplashScreen.java:372)
	at org.nschmidt.ldparteditor.main.LDPartEditor.main(LDPartEditor.java:39)
```

**Comment by nilsschmidt1337** [comment]
Closed with commit ba9e556.

**Comment by nilsschmidt1337** [comment]
Fix was wrong.

**Comment by nilsschmidt1337** [comment]
Closed with commit 934979a.
